### PR TITLE
Correct golint warnings (1)

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

### CODE REVIEW

**Comment by gliptak** [comment]
@kaidaguerre please review

**Comment by kaidaguerre** [comment]
Thanks for this @gliptak. Sorry for delayed response, been a little tied up with the new release. I will go through this today if possible.

File: ociinstaller/db.go [hunk 1]
```diff
@@ -12,7 +13,11 @@ import (
 // InstallDB :: Install Postgres files fom OCI image
 func InstallDB(ctx context.Context, dblocation string) (string, error) {
 	tempDir := NewTempDir(dblocation)
-	defer tempDir.Delete()
+	defer func() {
+		if err := tempDir.Delete(); err != nil {
```
Comment (kaidaguerre) [inline ociinstaller/db.go:17]: it does not work well to output text to the console during this phase, as we have progress spinners which don't really play well with console messages: 
image

Ultimately the solution to getting these kinds of warnings out would be for Steampipe to write a log-file. For now a decent compromise would be to include this as trace logging: 

```
log.Printf("[TRACE] Failed to delete temp dir '%s' after installing db files: %s", tempDir, err)
```

File: task/runner.go [hunk 2]
```diff
@@ -70,7 +71,9 @@ func (r *Runner) Run() {
 		}
 
 		// save the state - this updates the last checked time
-		r.currentState.Save()
+		if err := r.currentState.Save(); err != nil {
+			fmt.Println("Save currentState failed:", err)
```
Comment (kaidaguerre) [inline task/runner.go:75]:
```
utils.ShowWarning(fmt.Sprintf("Regular task runner failed to save state file: %s", err))
```

**Review by kaidaguerre — CHANGES_REQUESTED** [review]
Looks good!

A few minor tweaks requested.

File: task/runner.go (hunk 2 above)
Comment (gliptak) [reply task/runner.go:75]: updated

File: ociinstaller/db.go (hunk 1 above)
Comment (gliptak) [reply ociinstaller/db.go:17]: updated

File: db/db_local/local_db_client_connections.go [hunk 3]
```diff
@@ -21,6 +21,9 @@ import (
 // return whether any changes have been made
 func (c *LocalDbClient) refreshConnections(ctx context.Context) *steampipeconfig.RefreshConnectionResult {
 	res := &steampipeconfig.RefreshConnectionResult{}
+	if res.Error != nil {
```
[the remark this answers is not in the document]
Comment (gliptak) [reply db/db_local/local_db_client_connections.go]: deleted altogether

**Review by kaidaguerre — APPROVED** [review]
Looks good 👍

**Pull request merged**